test/integration/integration_test.go

```diff
@@ -68,8 +69,30 @@ func TestMain(m *testing.M) {
 	runner := util.MinishiftRunner{CommandPath: minishiftBinary}
 	if runner.IsCDK() {
 		godogTags += "~minishift-only"
+		isoName = "rhel"
+		fmt.Println("Test run using CDK binary with RHEL iso.")
 	} else {
 		godogTags += "~cdk-only"
+		switch os.Getenv("MINISHIFT_ISO_URL") {
+		case "b2d":
+			fmt.Println("Test run using Boot2Docker iso image.")
+			isoName = "b2d"
```

Comment:
ahh, that's nice. We use this to determine the ISO type. Works I guess while we use short names. Should be good enough. I was initially thinking of determining the ISO dynamically using the release info content of the VM.